composer.json

```diff
@@ -10,7 +10,7 @@
     "pimcore/asset-metadata-class-definitions": "<2.0 || > 2.0.99",
     "pimcore/data-hub-file-export": "<2.1 || > 2.1.99",
     "pimcore/data-hub-productsup": "<2.1 || > 2.1.99",
-    "pimcore/data-hub-simple-rest": "<3.0 || > 3.0.99",
+    "pimcore/data-hub-simple-rest": "<3.0.1 || > 3.0.99",
```

Comment:
@fashxp 3.0.1 is not released yet. Currently composer updates are not possible.

Reply:
Ups, then please revert and add a todo

Reply:
Done